onnxruntime/core/optimizer/graph_transformer_utils.cc

```diff
@@ -283,6 +286,11 @@ InlinedVector<std::unique_ptr<GraphTransformer>> GenerateTransformers(
       // The QDQFinalCleanupTransformer must run AFTER other transformers that fuse Q/DQ nodes. Otherwise, their
       // fusions might be prevented if this one removes a Q/DQ node too early.
       transformers.emplace_back(std::make_unique<QDQFinalCleanupTransformer>(enable_quant_qdq_cleanup));
+#ifdef USE_DML
+      // This transformer applies DML-specific fusions that go beyond what ORT offers by default
+      const InlinedHashSet<std::string_view> dml_ep = {onnxruntime::kDmlExecutionProvider};
+      transformers.emplace_back(std::make_unique<Dml::GraphTransformer>("DmlOperatorFusionTransformer", dml_ep));
```

Comment:
I am renaming this transformer as DmlOperatorFusionTransformer, so that we have clear distinction between the 2 transformers we are registering. One is DmlGraphFusionTransformer and the other is DmlOperatorFusionTransformer.